junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import csv
+import os
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from settings.common import ROOT_DIR
+
+from junction.base.constants import ProposalStatus
+from junction.conferences.models import Conference
+from junction.proposals.models import Proposal, ProposalSectionReviewerVoteValue
+from junction.proposals.permissions import is_conference_moderator
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--conference_slug",
+            default=None,
+            help="Enter the conference slug where to export reviewer votes from",
+        )
+
+        parser.add_argument("--user_id", default=None, help="Enter your user id")
+
+    def handle(self, *args, **options):
+
+        try:
+            conference = Conference.objects.get(slug=options.get("conference_slug"))
+            user = User.objects.get(id=options.get("user_id"))
+        except User.DoesNotExist:
+            self.stdout.write("Invalid user")
+            sys.exit(1)
+        except Conference.DoesNotExist:
+            self.stdout.write("Invalid conference")
+            sys.exit(1)
+
+        if not is_conference_moderator(user=user, conference=conference):
+            self.stdout.write("The user id is not a conference moderator")
+            sys.exit(1)
+
+        proposal_sections = conference.proposal_sections.all()
+        proposals_qs = Proposal.objects.select_related(
+            "proposal_type", "proposal_section", "conference", "author"
+        ).filter(conference=conference, status=ProposalStatus.PUBLIC)
+        proposals_qs = sorted(
+            proposals_qs, key=lambda x: x.get_reviewer_votes_sum(), reverse=True
+        )
+        proposal_vote_values = ProposalSectionReviewerVoteValue.objects.order_by(
+            "-vote_value"
+        )
+        vote_values_list = [v.vote_value for v in proposal_vote_values]
+        vote_values_desc = tuple(i.description for i in proposal_vote_values)
+        header = (
```

Comment:
Why is there an extra pair of brackets ?

Reply:
done